spec/controllers/assets_controller_spec.rb

```diff
@@ -220,23 +220,22 @@
     end
 
     describe "cache headers" do
+      let(:asset) { FactoryGirl.create(:asset) }
+
       it "sets the cache-control headers to 0 for an unscanned asset" do
-        asset = FactoryGirl.create(:asset, state: 'unscanned')
```

Comment:
Yeah. It's the default. I contemplated leaving the state set explicitly within the `let` block or introducing an `unscanned_asset` factory, but it felt like overkill. I'm happy if you want to look into doing that, but I think there are probably a bunch of other places where we'd need to use it.